src/fields/components/InternalField.tsx

```diff
@@ -74,10 +77,53 @@ export const InternalField = ({
           {errorMsg}
         </Text>
       )}
+
+      {/* When completed is false, whitespace is rendered */}
+      {completed !== undefined && (
+        <AnimationWrapper
+          displayStatus={completed}
+          isError={!!(error && errorMsg)}
+        >
+          <StatusWrapper mt={'8px'}>
```

Comment:
I tried to use flexProps on Box but it doesn't actually allow a gap of 4px, so I kept it as StatusWrapper. 
Unless we are happy to add '4px' to `type Spacing`?